src/pages/DispensingPage.js

```diff
@@ -63,7 +72,7 @@ const Dispensing = ({
           getCallback={getCellCallbacks}
           columns={columns}
           rowIndex={index}
-          onPress={editPatient}
+          onPress={dataSet === DATA_SET.PATIENTS ? editPatient : editPrescriber}
```

Comment:
would it be overkill to have a small method which held this logic? It's just that it is used a number of times which opens up the potential for the logic to be changed in one call and not in the others ( some time in the future when ideas change )

```
  const isPatient = () => !!DATA_SET.PATIENTS;
```

and 

```
          onPress={dataSet === isPatient ? editPatient : editPrescriber}
```

Reply:
25ed0ead2519bd3660e50ba222c17ec23487c1fd

nice! added it here, but did it slightly different because I'm difficult